test/Hooks.t.sol

```diff
@@ -168,8 +168,8 @@ contract HooksTest is Test, Deployers, GasSnapshot {
     }
 
     // hook validation
-    function testValidateHookAddressNoHooks(uint152 addr) public {
-        uint160 preAddr = uint160(uint256(addr));
+    function testValidateHookAddressNoHooks(uint160 addr) public {
+        uint160 preAddr = addr & uint160(0x007ffffFfffffffffFffffFFfFFFFFFffFFfFFff);
```

Comment:
I have a change to this file on this branch that I think will make updating these tests a lot easier when we introduce new flags. See here -(just flagging) since we'd have conflicts. https://github.com/Uniswap/v4-core/blob/access-lock/test/Hooks.t.sol#L32

Reply:
oh niceeeeee, great change